project_event/i18n/fr.po

```diff
@@ -1423,13 +1430,13 @@ msgid "res.partner.sector"
 msgstr "res.partner.sector"
 
 #. module: project_event
-#: code:addons/project_event/models/project_task.py:380
+#: code:addons/project_event/models/project_task.py:381
 #, python-format
 msgid "this resource is not bookable"
```

Comment:
Change this to This (Sentence case)

Reply:
It comes after : so i have to change the fr to lower case